package/MDAnalysis/topology/PDBParser.py

```diff
@@ -515,3 +540,131 @@ def _parse_conect(conect):
     bond_atoms = (int(conect[11 + i * 5: 16 + i * 5]) for i in
                   range(n_bond_atoms))
     return atom_id, bond_atoms
+
+
+def fetch_pdb(
+    pdb_ids=None,
+    cache_path=None,
+    progressbar=False,
+    file_format="pdb.gz",
+):
+    """
+    Download one or more PDB files from the RCSB Protein Data Bank and cache
+    them locally.
+
+    Given one or multiple PDB IDs, downloads the corresponding structure files
+    format and stores them in a local cache directory. If files are cached on
+    disk, fetch_pdb() will skip the download and use the cached version instead.
+
+    Returns the path(s) as a string to the downloaded file(s).
+
+    Parameters
+    ----------
+    pdb_ids : str or sequence of str
+        A single PDB ID as a string, or a sequence of PDB IDs to fetch.
+    cache_path : str or pathlib.Path
+        Directory where downloaded file(s) will be cached.
```

Comment:
Explain here what ``None`` does
```suggestion
        Directory where downloaded file(s) will be cached.
        The default ``None`` uses the :mod:`pooch` default cache with 
        project name :data:`DEFAULT_CACHE_NAME_DOWNLOADER`.
```
(assuming you also document DEFAULT_CACHE_NAME_DOWNLOADER as suggested)